### PR TITLE
User agent fix

### PR DESCRIPTION
**Pull request by Scollaco**
# What
Fixes an issue where user-agent header was being overwritten with a default value,  making some `Completed Checkout` events be shown  as `client_type = web`.

# Why
https://trello.com/c/pl9yHIQ0/543-native-clienttype-detection

### CODE REVIEW

**Comment by Scollaco** [comment]
[Native client_type detection](https://trello.com/c/pl9yHIQ0/543-native-clienttype-detection)

File: KsApi/ServiceType.swift [hunk 1]
```diff
@@ -407,6 +407,12 @@ extension ServiceType {
     headers["Authorization"] = self.authorizationHeader
     headers["Kickstarter-App-Id"] = self.appId
     headers["Kickstarter-iOS-App"] = self.buildVersion
+    headers["User-Agent"] = Self.userAgent
```
Comment (eoji) [inline KsApi/ServiceType.swift]: what's the difference between `self` and `Self`? 🕵🏾‍♀️

Comment (Scollaco) [reply KsApi/ServiceType.swift]: `Self` (capitalized) refers to the type that conforms to the protocol. (In this case, `ServiceType`). While `self` is equivalent to `this` in Java.

Comment (eoji) [reply KsApi/ServiceType.swift]: amazing

**Pull request merged**